ports/stm/Makefile

```diff
@@ -110,6 +110,8 @@ CFLAGS += $(MCU_FLAGS_$(MCU_SERIES))
 # Select HAL file for distribution via mpconfigport
 CFLAGS += -DSTM32_HAL_H='<stm32$(MCU_SERIES_LOWER)xx_hal.h>'
 
+CFLAGS += -DSTM32_SERIES='"stm32$(MCU_SERIES_LOWER)"'
```

Comment:
Can you make this STM32_SERIES_LOWER, to maintain clarity? Case doesn't matter that often but it sometimes comes up between mac/linux builds, and I'd rather not have two STM32_SERIES variables that have opposite case.